scripts/manage_study.py

```diff
@@ -84,6 +93,18 @@ def login(manager=None, dry_run=False, api_base=None, verbose=False):
                       api_base=api_base)
     return(manager)
 
+def validate_metadata_file(convention_path, metadata_path):
+    with open(convention_path) as f:
+        convention = json.load(f)
+    metadata = CellMetadata(metadata_path, '', '')
+    print(f'Validating {metadata_path}')
+    format_valid = metadata.validate_format()
```

Comment:
format_valid is unnecessary here. I have a note to self to clean this up in scp-ingest-pipeline

Reply:
Addressed in 9ef1516.